### PR TITLE
Add unit test of StandalonePersistRepositoryConfigurationYamlSwapper.

### PR DESCRIPTION
**Pull request by fwhdzh**
For #12407.

Changes proposed in this pull request:
- add unit test of StandalonePersistRepositoryConfigurationYamlSwapper.

### CODE REVIEW

File: shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/standalone/StandalonePersistRepositoryConfigurationYamlSwapperTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+public final class StandalonePersistRepositoryConfigurationYamlSwapperTest {
+
+    @Test
+    public void assertSwapToYamlConfiguration() {
+        final String testType = "testType";
+        Properties testProps = new Properties();
+        StandalonePersistRepositoryConfiguration standalonePersistRepositoryConfiguration = new StandalonePersistRepositoryConfiguration(testType, testProps);
+        StandalonePersistRepositoryConfigurationYamlSwapper standalonePersistRepositoryConfigurationYamlSwapper = new StandalonePersistRepositoryConfigurationYamlSwapper();
+        YamlPersistRepositoryConfiguration result = standalonePersistRepositoryConfigurationYamlSwapper.swapToYamlConfiguration(standalonePersistRepositoryConfiguration);
```
Comment (menghaoranss) [inline shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/standalone/StandalonePersistRepositoryConfigurationYamlSwapperTest.java:38]: result -> actual

**Review by menghaoranss — CHANGES_REQUESTED** [review]

File: shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/standalone/StandalonePersistRepositoryConfigurationYamlSwapperTest.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import static org.junit.Assert.assertThat;
+
+public final class StandalonePersistRepositoryConfigurationYamlSwapperTest {
+
+    @Test
+    public void assertSwapToYamlConfiguration() {
+        final String testType = "testType";
+        Properties testProps = new Properties();
+        StandalonePersistRepositoryConfiguration standalonePersistRepositoryConfiguration = new StandalonePersistRepositoryConfiguration(testType, testProps);
+        StandalonePersistRepositoryConfigurationYamlSwapper standalonePersistRepositoryConfigurationYamlSwapper = new StandalonePersistRepositoryConfigurationYamlSwapper();
```
[the remark this answers is not in the document]
Comment (fwhdzh) [reply shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/standalone/StandalonePersistRepositoryConfigurationYamlSwapperTest.java:37]: Emm...in other issue, another supervisor asked me to use local variables as possible. 
"Minimize the use of variables. The life cycle of the constant defined in the class is bound to the bytecode file.If xxx is only used in methods, use local variables. The life cycle of the variable is bound to the method." (issue 12159)
Should I use local variable or member variable?

[the remark this answers is not in the document]
Comment (menghaoranss) [reply shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/standalone/StandalonePersistRepositoryConfigurationYamlSwapperTest.java:37]: Swapper is just a stateless class, no need to repeat the definition.

File: shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/standalone/StandalonePersistRepositoryConfigurationYamlSwapperTest.java (hunk 1 above)
Comment (fwhdzh) [reply shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/standalone/StandalonePersistRepositoryConfigurationYamlSwapperTest.java]: have fixed

File: shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/standalone/StandalonePersistRepositoryConfigurationYamlSwapperTest.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (fwhdzh) [reply shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/standalone/StandalonePersistRepositoryConfigurationYamlSwapperTest.java:38]: have fixed

File: shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/standalone/StandalonePersistRepositoryConfigurationYamlSwapperTest.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public final class StandalonePersistRepositoryConfigurationYamlSwapperTest {
+
+    public static final String TEST_TYPE = "TEST_TYPE";
+
+    @Getter
+    private StandalonePersistRepositoryConfigurationYamlSwapper standalonePersistRepositoryConfigurationYamlSwapper = new StandalonePersistRepositoryConfigurationYamlSwapper();
```
Comment (menghaoranss) [inline shardingsphere-mode/shardingsphere-mode-type/shardingsphere-standalone-mode/shardingsphere-standalone-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/standalone/StandalonePersistRepositoryConfigurationYamlSwapperTest.java:36]: standalonePersistRepositoryConfigurationYamlSwapper -> swapper, and why need @Getter?

**Review by menghaoranss — CHANGES_REQUESTED** [review]

**Review by menghaoranss — APPROVED** [review]

**Pull request merged**